module_3/gift_certificates_system/model/src/main/java/com/epam/esm/util/DtoRegexValidator.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.esm.util;
+
+public final class DtoRegexValidator {
+    // User DTO
+    public static final String USER_LOGIN = "^[\\p{Ll}\\p{Lu}]{4,50}$";
+    public static final String USER_NAME = "^\\p{Lu}\\p{Ll}{1,50}$";
+
+    // Tag DTO
+    public static final String TAG_NAME = "^\\p{Lu}\\p{Ll}{1,50}$";
+
+    // Gift certificate DTO
```

Comment:
There's no need in comments, variable names are verbose enough

Reply:
I use it for clarity, it's convenient.